### PR TITLE
fix: version policy filter

### PR DESCRIPTION
**Pull request by pranav-bhuchhada**

### CODE REVIEW

File: src/libs/utils.js [hunk 1]
```diff
@@ -74,6 +67,13 @@ const getAllChanges = async ({ rushChangePath, packagePaths, options = {} }) =>
       }
     }
   }
+  // Filter projects by version policy
+  if (options.versionPolicy && options.versionPolicy != "") {
```
Comment (pkrawiec-advanced) [inline src/libs/utils.js:71]: Is the second condition needed?

**Review by pkrawiec-advanced — APPROVED** [review]

File: src/libs/utils.js (hunk 1 above)
Comment (pranav-bhuchhada) [reply src/libs/utils.js:70]: For jest unit tests it required, because when we run unit test `options.versionPolicy` will be `undefined`

Comment (pkrawiec-advanced) [reply src/libs/utils.js:70]: `undefined` should also be picked by the first condition.

Comment (pranav-bhuchhada) [reply src/libs/utils.js:70]: Ah you are right, I got confused, Will update it.

**Pull request merged**